DQMOffline/Trigger/python/HiggsMonitoring_Client_cff.py

```diff
@@ -1,4 +1,18 @@
 import FWCore.ParameterSet.Config as cms
 
-higgsClient = cms.Sequence(
+diphotonEfficiency = cms.EDAnalyzer("DQMGenericClient",
+    subDirs        = cms.untracked.vstring("HLT/Higgs/*"),
+    verbose        = cms.untracked.uint32(0), # Set to 2 for all messages
+    resolution     = cms.vstring(),
+    efficiency     = cms.vstring(
+        "eff_diphoton         'Photon turnON;             Photon pt [GeV]; efficiency'     photon_pt_numerator          photon_pt_denominator",
+        "eff_diphoton_variable 'Photon turnON;            Photon pt [GeV]; efficiency'     photon_pt_variable_numerator photon_pt_variable_denominator",
```

Comment:
Sorry, I will add the eta plot.  You want also the 2-d eta vs phi efficiency?

The mass plot requires a few changes to the .cc file (discussed in a later comment).